src/tiled/tilestampmanager.cpp

```diff
@@ -137,6 +137,7 @@ static TileStamp stampFromContext(AbstractTool *selectedTool)
         copyMap->addTilesets(copy->usedTilesets());
 
         copyMap->setRenderOrder(map->renderOrder());
+        copyMap->setObjectAlignment(map->objectAlignment());
```

Comment:
The tile stamps only contain tile layers, so this property is irrelevant here.

Reply:
Okay. I was just doing due diligence. I don't actually know what tile stamps are! XD